src/wms/src/main/java/org/geoserver/wms/WMSInfo.java

```diff
@@ -200,4 +200,13 @@ default void setMaxRequestedDimensionValues(int maxRequestedDimensionValues) {
 
     /** Sets the remote style timeout in milliseconds * */
     void setRemoteStyleTimeout(int time);
+
+    default boolean isIncludeDefaultGroupStyleInCapabilities() {
```

Comment:
Too long. Maybe:

```suggestion
    default boolean isDefaultGroupStyleEnabled() {
```

Tried hard to make it shorter than that (still pretty long), came out empty so far.... suggestions welcomed :-D

Reply:
fixed

Reply:
fixed